sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -3111,6 +3132,12 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val isDeltaLowShuffleMergeEnabled: Boolean = get(ENABLE_DELTA_LOW_SHUFFLE_MERGE)
 
+  lazy val isTagLoreIdEnabled: Boolean = get(TAG_LORE_ID_ENABLED)
+
+  lazy val loreDumpIds: Option[String] = get(LORE_DUMP_IDS)
```

Comment:
As I mentioned before when suggesting lazy vals, I don't think we want to cache the string here.  We're caching the lookup, but we're not caching the parse.  The parse is much more expensive than the lookup, but we're reparsing this every time when we go through tagging.
```suggestion
  lazy val loreDumpIds: Map[LoreId, OutputLoreId] = get(LORE_DUMP_IDS).map(OutputLoreId.parse).getOrElse(Map.empty)
```
By using an empty map by default, code that just wants to know if a lore ID is being tracked can just use the map directly rather than needing to worry about whether there's a map at all or not.  If code wants to know whether the config was set, they can check for an empty map.